src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -970,6 +983,14 @@ public void run() {
         }
     }
 
+    private void runCMD(String cmd) throws IOException, InterruptedException {
+        Process p = Runtime.getRuntime().exec(cmd);
+        p.waitFor(5, TimeUnit.SECONDS);
+        if (p.exitValue() != 0) {
+            debug("Error running command:" + cmd + ", return value=" + p.exitValue());
```

Comment:
Should this log as error?  Or do we expect that some of the commands will fail?

Reply:
I agree, the user may need to adjust the directories.